index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!-- If the browser is lower than or equal to IE 9, redirect to the old page.
+        TODO verificar outros navegadores
+        TODO colocar a página correta de redirecionamento (quando for para produção, 
+            criar um link para a página antiga e.g. matrusp/old, matrusp/v1) -->
+    <!--[if lte IE 9 ]>
+        <meta http-equiv="refresh" content="0; url=http://bcc.ime.usp.br/matrusp/" />
+    <![endif]-->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>MATRUSP</title>
+    
+    <link rel="stylesheet" href="styles/css/application.css">
+    
+</head>
+
+<body>
+    <div id="page-container">
+        <header>
+            <span id="logo">MATRUSP</span>
+            <ul id="icons">
+                <li class="icon" data-description="Exportar para o calendario do Google">
+                    <div class="hover-mask"></div>
+                    <img src="images/ic_calendar.png" alt="">
+                </li>
+                <li class="icon" data-description="Imprimir">
+                    <div class="hover-mask"></div>
+                    <img src="images/ic_print.png" alt="">
+                </li>
+                <li class="icon" data-description="Enviar por E-mail">
+                    <div class="hover-mask"></div>
+                    <img src="images/ic_email.png" alt="">
+                </li>
+                <li class="icon" data-description="Compartilhar pelo Facebook">
+                    <div class="hover-mask"></div>
+                    <img src="images/ic_facebook.png" alt="">
+                </li>
+                <li class="icon" data-description="Fazer download">
+                    <div class="hover-mask"></div>
+                    <img src="images/ic_download.png" alt="">
+                </li>
+            </ul>
+        </header>
+        <div id="main">
+            <div id="lecture-schedule">
+                <div class="column time">
+                    <div class="hour">06:00</div>
+                    <div class="hour">07:00</div>
+                    <div class="hour">08:00</div>
+                    <div class="hour">09:00</div>
+                    <div class="hour">10:00</div>
+                    <div class="hour">11:00</div>
+                    <div class="hour">12:00</div>
+                    <div class="hour">13:00</div>
+                    <div class="hour">14:00</div>
+                    <div class="hour">15:00</div>
+                    <div class="hour">16:00</div>
+                    <div class="hour">17:00</div>
+                    <div class="hour">18:00</div>
+                    <div class="hour">19:00</div>
+                    <div class="hour">20:00</div>
+                    <div class="hour">21:00</div>
+                    <div class="hour">22:00</div>
+                    <div class="hour">23:00</div>
+                    <div class="hour">00:00</div>
```

Comment:
Localization: We should generate hours of the day dynamically. Some cultures use AM/PM, others use 24h.